lib/templates/html/default.ts

```diff
@@ -0,0 +1,102 @@
+// DO NOT EDIT THIS FILE
```

Comment:
maybe better to not commit this file a make a npm task at build time to generate it ?

Reply:
what's the benefit in not committing it?

Reply:
- does not appear in diffs
- it's not immediately clear if it's aligned to the lastest template